pwa/components/common/text/subtitle.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+type SubtitleProps = {
+  text: string;
+  level?: 1 | 2;
+};
+
+export const Subtitle: React.FC<SubtitleProps> = ({ text, level = 1 }) => {
+  const Tag = level === 1 ? 'h3' : 'h4';
+
+  return <Tag className={`text-${level === 1 ? 'lg' : 'md'} font-semibold mt-4`}>{text}</Tag>;
```

Comment:
Use tailwind `heading` classes instead of the `semibold`

Reply:
There is no font weight modification without font-semibold
![image](https://github.com/ReyalWesley/httpCache/assets/78682200/813140ad-e89a-4ff4-bafb-c91809d515dd)

Reply:
Here are some documentations about that.

https://daisyui.com/docs/layout-and-typography
https://tailwindcss.com/docs/typography-plugin
https://www.phuctm97.com/blog/misundertand-tailwind-typography-cost-me-2h